lib/loaders/saved_artwork.js

```diff
@@ -0,0 +1,20 @@
+import DataLoader from 'dataloader';
+import gravity from './gravity';
+import { map, find, extend } from 'lodash';
+
+export default (userId, accessToken) => {
+  return new DataLoader(ids => {
+    return gravity.with(accessToken)('collection/saved-artwork/artworks', {
+      user_id: userId,
+      artworks: ids,
+      private: true,
+    }).then((results) => {
+      const parsedResults = map(ids, (id) => {
+        const match = find(results, { id });
+        if (match) return extend(match, { is_saved: true });
+        return { id, is_saved: false };
```

Comment:
DataLoader requires that you send back an array of the same size to resolve the promise, but gravity currently just doesn't include the artwork in the response if the artwork isn't saved.